lib/jekyll/admin/configuration/handler.rb

```diff
@@ -0,0 +1,22 @@
+require 'yaml'
+module Jekyll
+  module Admin
+    module Configuration
```

Comment:
Take a look at https://github.com/jekyll/jekyll/blob/master/lib/jekyll/configuration.rb. I think we'll want the raw config file, but Jekyll has some helpers to help you find its path and safely read it across different formats.